operation/fhir-operation-bulkdata/src/main/java/com/ibm/fhir/operation/bulkdata/BulkDataResourceProvider.java

```diff
@@ -1,16 +0,0 @@
-/*
- * (C) Copyright IBM Corp. 2020
- *
- * SPDX-License-Identifier: Apache-2.0
- */
-
-package com.ibm.fhir.operation.bulkdata;
-
-import com.ibm.fhir.registry.util.PackageRegistryResourceProvider;
-
-public class BulkDataResourceProvider extends PackageRegistryResourceProvider {
```

Comment:
I believe this registry resource provider is what we were using to advertise our support of the bulkdata spec in our CapabilityStatement.  Are you still advertising that capability, but in a new way?
See https://github.com/IBM/FHIR/issues/1315 for more info.

Reply:
ah, now I see it was just moved to `ResourceProvider` under subpackage `provider`.